src/main/resources/static/mini-manage-api-stubs.yml

```diff
@@ -359,6 +377,167 @@ paths:
           description: unexpected error
           content: {}
       x-codegen-request-body-name: body
+  /reference-data/departure-reasons:
+    get:
+      tags:
+        - Reference Data
+      summary: Lists all departure reasons
+      responses:
+        200:
+          description: successful operation
+          content:
+            'application/json':
+              schema:
+                type: array
+                items:
+                  $ref: '#/components/schemas/DepartureReason'
+        401:
+          description: not authenticated
+          content: { }
```

Comment:
Minor point but just ommitting the content key here achieves the same result as providing an empty map

Reply:
although I notice this is how it's done elsewhere in the file, so maybe one for a seperate PR

Reply:
👍 thanks @brownbearsoftware

Reply:
sorry forgot to actually approve it!